src/Microsoft.AspNetCore.Mvc.Abstractions/Filters/IFilterProvider.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Microsoft.AspNetCore.Mvc.Filters
 {
+    /// <summary>
+    /// A <see cref="FilterItem"/> provider.
```

Comment:
I'd mention here that a filter provider can "provide an executable filter associated with an IFilterMetadata by setting FilterItem.Filter, or add FilterItems to add new filters"